### ISSUE TITLE
Question about loss = criterion(output, y_batch)

### ISSUE THREAD

**Issue by pleasurepants** [jeya-maria-jose/Medical-Transformer#63]
Sry to interupt, but when i was using my own dataset to train the model. I've found a question about following. Can u explain that for me?
![image](https://user-images.githubusercontent.com/80297871/146730768-8edf423d-1c74-4253-b496-89f5535ce792.png)

**Comment by jeya-maria-jose** [comment]
The dimension of your tensor should be of the form NCHW and not NHWC while using cross-entropy loss.
